core/pta/tests/misc.c

```diff
@@ -548,14 +550,26 @@ static int self_test_nex_malloc(void)
 	return 0;
 }
 #endif
+
+static int self_test_dt_driver(void)
+{
+	if (IS_ENABLED(CFG_DT_DRIVER_EMBEDDED_TEST)) {
+		LOG("dt_driver test:");
+		if (dt_test_status())
+			return -1;
+	}
+
+	return 0;
+}
+
 /* exported entry points for some basic test */
 TEE_Result core_self_tests(uint32_t nParamTypes __unused,
 		TEE_Param pParams[TEE_NUM_PARAMS] __unused)
 {
 	if (self_test_mul_signed_overflow() || self_test_add_overflow() ||
 	    self_test_sub_overflow() || self_test_mul_unsigned_overflow() ||
 	    self_test_division() || self_test_malloc() ||
-	    self_test_nex_malloc()) {
+	    self_test_nex_malloc() || self_test_dt_driver()) {
```

Comment:
I'd rather have this in a separate test case.
I suppose we can keep it in 1001, but via another subcase.

Reply:
ok

Reply:
Is https://github.com/OP-TEE/optee_test/pull/591 what you had in mind?

Reply:
Yes, thanks.